ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -85,7 +88,7 @@ static int S3ObjectListSingle(
 static int S3ObjectListFree(WT_FILE_SYSTEM *, WT_SESSION *, char **, uint32_t);
 
 /*
- * S3Exist --
+ *   --
```

Comment:
? :)
Lets keep the name of the function we are writing about.

Reply:
Haha yes! I just put these back in. Accident!